src/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -44,226 +44,239 @@ const InstanceDetailPanel: FC = () => {
                     to={`/ui/project/${instance.project}/instance/${instance.name}/snapshots`}
                   >
-                    Snapshots
+                    {`View all (${instance.snapshots.length})`}
                   </Link>
-                </h3>
-              </th>
-            </tr>
-            {instance.snapshots?.length ? (
-              <>
-                {instance.snapshots
-                  .slice()
-                  .sort((snap1, snap2) => {
-                    const a = snap1.created_at;
-                    const b = snap2.created_at;
-                    return a > b ? -1 : a < b ? 1 : 0;
-                  })
-                  .slice(0, RECENT_SNAPSHOT_LIMIT)
-                  .map((snapshot) => (
-                    <tr key={snapshot.name} className="u-no-border">
-                      <th
-                        title={snapshot.name}
-                        className="snapshot-name u-truncate"
-                      >
-                        <ItemName item={snapshot} />
-                      </th>
-                      <td className="u-text--muted">
-                        <i>{isoTimeToString(snapshot.created_at)}</i>
-                      </td>
-                    </tr>
-                  ))}
-                {instance.snapshots.length > RECENT_SNAPSHOT_LIMIT && (
-                  <tr>
-                    <td colSpan={2}>
-                      <Link
-                        to={`/ui/project/${instance.project}/instance/${instance.name}/snapshots`}
-                      >
-                        {`View all (${instance.snapshots.length})`}
-                      </Link>
-                    </td>
-                  </tr>
-                )}
-              </>
-            ) : (
-              <tr>
-                <td colSpan={2}>
-                  <p className="no-snapshots">
-                    No snapshots found.
-                    <br />
-                    <Link
-                      to={`/ui/project/${instance.project}/instance/${instance.name}/snapshots`}
-                    >
-                      Manage instance snapshots
-                    </Link>
-                  </p>
                 </td>
               </tr>
             )}
-          </tbody>
-        </table>
-      )}
-    </DetailPanel>
+          </>
+        ) : (
+          <tr>
+            <td colSpan={2}>
+              <p className="no-snapshots">
+                No snapshots found.
+                <br />
+                <Link
+                  to={`/ui/project/${instance.project}/instance/${instance.name}/snapshots`}
+                >
+                  Manage instance snapshots
+                </Link>
+              </p>
+            </td>
+          </tr>
+        )}
+      </tbody>
+    </table>
+  );
+
+  return (
+    <SidePanel
+      loading={isLoading}
+      hasError={!instance}
+      className="detail-panel instance-detail-panel"
+    >
+      <SidePanel.HeaderStickyWrapper>
```

Comment:
This is a wired name. Maybe just `SidePanel.Sticky`, or maybe you have a better idea?

Reply:
Sticky sounds good :+1: